ui/components/app/confirm/footer/footer.tsx

```diff
@@ -1,5 +1,40 @@
 import React from 'react';
+import {
+  Box,
+  Button,
+  ButtonSize,
+  ButtonVariant,
+} from '../../../component-library';
+import {
+  BlockSize,
+  Display,
+  FlexDirection,
+} from '../../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
 
-const Footer = () => <div>CONFIRM FOOTER COMPONENT</div>;
+const Footer = () => {
+  const t = useI18nContext();
+  return (
+    <Box
+      display={Display.Flex}
+      flexDirection={FlexDirection.Row}
+      gap={4}
+      padding={4}
+      width={BlockSize.Full}
+    >
+      <Button
+        block
+        data-testid="cancel-button"
+        variant={ButtonVariant.Secondary}
+        size={ButtonSize.Lg}
+      >
+        {t('cancel')}
+      </Button>
+      <Button size={ButtonSize.Lg} block data-testid="confirm-button">
+        {t('confirm')}
```

Comment:
will we be supporting different primary button text?

Reply:
@jpuri mentioned this confirm footer will only be used in one place, hence why I removed the props.